test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageSearchServiceTests.cs

```diff
@@ -426,10 +428,74 @@ public async Task CreatePackageFeedAsync_WithTransitiveOriginsExpFlag_OnlyInstal
                 sourceRepositories: new List<SourceRepository>() { _sourceRepository },
                 cancellationToken: CancellationToken.None);
 
+            // Assert
             Assert.IsType(expectedFeedType, main);
             Assert.Null(recommender);
         }
 
+        [Fact]
+        public async Task CreatePackageFeedAsync_ProjectPMUIInstalledTab_EmitsCounterfactualTelemetryAsync()
+        {
+            // Arrange
+            var telemetrySession = new Mock<ITelemetrySession>();
+            var telemetryEvents = new ConcurrentQueue<TelemetryEvent>();
+            telemetrySession
+                .Setup(x => x.PostEvent(It.IsAny<TelemetryEvent>()))
+                .Callback<TelemetryEvent>(x => telemetryEvents.Enqueue(x));
+            TelemetryActivity.NuGetTelemetryService = new NuGetVSTelemetryService(telemetrySession.Object);
+
+            using NuGetPackageSearchService searchService = SetupSearchService();
+            CounterfactualLogger.PMUITransitiveDependencies.Reset();
+
+            // Act
+            _ = await searchService.CreatePackageFeedAsync(
+                projectContextInfos: _projects,
+                targetFrameworks: new List<string>() { "net45" },
+                itemFilter: ItemFilter.Installed,
+                isSolution: false,
+                recommendPackages: It.IsAny<bool>(),
+                sourceRepositories: new List<SourceRepository>() { _sourceRepository },
+                cancellationToken: CancellationToken.None);
+
+            // Assert
+            Assert.Contains(telemetryEvents, evt => evt.Name == CounterfactualLogger.PMUITransitiveDependencies.EventName);
+        }
+
+        [Theory]
+        [InlineData(ItemFilter.All, true)]
+        [InlineData(ItemFilter.Installed, true)]
+        [InlineData(ItemFilter.UpdatesAvailable, true)]
+        [InlineData(ItemFilter.Consolidate, true)]
+        [InlineData(ItemFilter.All, false)]
```

Comment:
Is "ItemFilter.Installed, false" missing or it's intentional?

Reply:
Intentional. Might add a comment.